CouchbaseLite.xcodeproj/project.pbxproj

```diff
@@ -122,6 +122,12 @@
 		1AAB2787227793E50037A880 /* CBLConflict.m in Sources */ = {isa = PBXBuildFile; fileRef = 1AAB273C22739EBB0037A880 /* CBLConflict.m */; };
 		1AAF6372226A8B060016754C /* QueryTest+Meta.m in Sources */ = {isa = PBXBuildFile; fileRef = 1AAF6371226A8B060016754C /* QueryTest+Meta.m */; };
 		1AAF6383226A8DBF0016754C /* QueryTest+Join.m in Sources */ = {isa = PBXBuildFile; fileRef = 1AAF6382226A8DBF0016754C /* QueryTest+Join.m */; };
+		1AC3E443294C977800156AE0 /* CBLDNSService.h in Headers */ = {isa = PBXBuildFile; fileRef = 1AC3E436294C977700156AE0 /* CBLDNSService.h */; };
+		1AC3E444294C977800156AE0 /* CBLDNSService.h in Headers */ = {isa = PBXBuildFile; fileRef = 1AC3E436294C977700156AE0 /* CBLDNSService.h */; };
+		1AC3E445294C977800156AE0 /* CBLDNSService.mm in Sources */ = {isa = PBXBuildFile; fileRef = 1AC3E442294C977700156AE0 /* CBLDNSService.mm */; };
+		1AC3E446294C977800156AE0 /* CBLDNSService.mm in Sources */ = {isa = PBXBuildFile; fileRef = 1AC3E442294C977700156AE0 /* CBLDNSService.mm */; };
+		1AC3E447294C979200156AE0 /* CBLDNSService.mm in Sources */ = {isa = PBXBuildFile; fileRef = 1AC3E442294C977700156AE0 /* CBLDNSService.mm */; };
+		1AC3E448294C979400156AE0 /* CBLDNSService.mm in Sources */ = {isa = PBXBuildFile; fileRef = 1AC3E442294C977700156AE0 /* CBLDNSService.mm */; };
```

Comment:
Just noticed the double record here for CBLDNSService.mm.

Reply:
Those four references might be for the '.mm' file included in 4 targets(CBL_Swift, CBL_ObjC, CBL_EE_ObjC, CBL_EE_Swift)? Same is for below `Foundation+CBL.mm`?

Reply:
I do not know how this actually works as there are exactly the same.

Reply:
I tried remove and added back, it's same 4 file references

Reply:
1AC3E445294C977800156AE0 - CBL_Swift
1AC3E446294C977800156AE0 - CBL_EE_Swift
1AC3E447294C979200156AE0 - CBL_ObjC
1AC3E448294C979400156AE0 - CBL_EE_ObjC

First ID differs and points to each target